### DISCUSSION TITLE
Theming / Layouts

### DISCUSSION THREAD

**Discussion by AndiLeni** [hydephp/hyde#225] [Q&A] [answered]
Hello,

I wanted to know, how I might adjust the default theme or how to ship my own design.
The resources/view folder seems to be for this kind of adjustments, but it is empty after the installation?
Where are the default templates located?

Maybe it would make sense to have the default layouts in the resources/view folder to allow new users a quicker start?
And a page in the documentation for this specific part would help.

Thanks and kind regards

**Comment by janar** [comment, marked as answer]
Answering part of the question. They can be published into your own project: 

`php hyde publish:views`

In docs: 
https://hydephp.com/docs/1.x/customization#blade-views
[publish:views](https://hydephp.com/docs/1.x/console-commands#publish-views)

They are located in: 
https://github.com/hydephp/framework/tree/master/resources/views

**Reply by AndiLeni** [reply]
Thanks for pointing to these links!
So it is intentional to not include them by default as far as I see. 
Should the files in `resources/views/vendor/hyde` be edited directly, or should they be copied into `resources/views`?
And my own views should also go into `resources/views` if I am correct?

**Reply by emmadesilva** [reply]
Indeed, the files in `resources/views/vendor/hyde` may be edited directly!

**Reply by timgavin** [reply]
Would seem to make sense if we could create our own views that would automatically overwrite the defaults. Say, if we created a `/resources/views/layouts/app.blade.php` file, Hyde would consider that our app view and disregard Hyde's defaults. The way it is now is somewhat confusing.

With that said, I only discovered Hyde 15 minutes ago and am getting the impression this is exactly what I've been looking for! :)

**Reply by emmadesilva** [reply]
Hey! Thanks for checking out Hyde! 

Actually, the way you describe it, this sounds exactly like the HydePHP design:

> Say, if we created a /resources/views/layouts/app.blade.php file, Hyde would consider that our app view and disregard Hyde's defaults. 

I'm gonna convert this to an issue for implementation! Possibly this will happen in v2.x but we'll see. Thanks again for the detailed feedback and for trying Hyde!

Edit: Tracked in https://github.com/hydephp/hyde/issues/258
